server-spi/src/main/java/org/keycloak/models/ClientProvider.java

```diff
@@ -20,30 +20,87 @@
 import org.keycloak.storage.client.ClientLookupProvider;
 
 import java.util.List;
-import java.util.Set;
 
 /**
+ * Provider of the client records.
+ *
  * @author <a href="mailto:bill@burkecentral.com">Bill Burke</a>
```

Comment:
We should change to author as well.

Reply:
I don't think we should in this case - Bill is the original author so I'd rather keep it.